cpp/daal/include/services/env_detect.h

```diff
@@ -42,11 +43,16 @@ namespace daal
  */
 enum CpuType
 {
+#ifdef __ARM_ARCH
+    sve         = 0, /*!< ARM processors based on Arm's Scalable Vector Extension (SVE) */
```

Comment:
are there plans to bring NEON instructions? Per my understanding sve is not the oldest one - if there plans to bring older ISA then it worth set SVE higher number